dev/github/helper.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# This script is called by both run_checks.sh and run_tests.sh to perform the initial build process
+
+#
+# This script is run from inside the Docker container
+#
+set -ex
+
+if [ -n "${ALLUXIO_GIT_CLEAN}" ]
+then
+  git clean -fdx
+fi
+
+mvn_args=""
+if [ -n "${ALLUXIO_MVN_RUNTOEND}" ]
+then
+  mvn_args+=" -fn -DfailIfNoTests=false --fail-at-end"
+fi
+
+export MAVEN_OPTS="-Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss.SSS"
+
+# Always use java 8 to compile the source code
+JAVA_HOME_BACKUP=${JAVA_HOME}
+PATH_BACKUP=${PATH}
+JAVA_HOME=/usr/local/openjdk-8
+PATH=$JAVA_HOME/bin:$PATH
+mvn -Duser.home=/home/jenkins -T 4C clean install -Pdeveloper -Dfindbugs.skip -Dcheckstyle.skip -DskipTests -Dmaven.javadoc.skip \
```

Comment:
This is for the checkstyle and findbugs task, right? This is also skipping checkstyle, findbugs, etc. Is this correct? I am a bit confused.

This is an output for this PR's checkstyle check. https://pipelines.actions.githubusercontent.com/110HvfICxw4TXSOnsd72jk32naeMsNgZF0xPcAxH5PwqdiBHUN/_apis/pipelines/1/runs/106/signedlogcontent/3?urlExpires=2020-09-30T17%3A18%3A37.2939398Z&urlSigningMethod=HMACV1&urlSignature=y%2B13RULZz8azm5yA9KZIWAnIzZTiBsg0JVWduMCxaDQ%3D 

However, in that output, I see
```
2020-09-30T08:20:56.6182722Z + mvn -Duser.home=/home/jenkins -T 4C clean install -Pdeveloper -Dfindbugs.skip -Dcheckstyle.skip -DskipTests -Dmaven.javadoc.skip -Dlicense.skip -Dsurefire.forkCount=2 -fn -DfailIfNoTests=false --fail-at-end
```

This looks like all the checks are skipped?

Reply:
Oh wow you are right. I accidentally dropped it while doing the refactor during the review process. I ended up undoing the refactor and moved helper.sh into the respective callers and modified the findbug/checkstyle skip parts.

Reply:
Fixed now.

Reply:
cool, thanks! The mvn command looks correct now.